### PR TITLE
Napp management refactor

### PR DESCRIPTION
**Pull request by diraol**
- Adds the NApps directory into the system path, so we can access the napps as python libraries;
 - Improve the "Load Napp" method;
 - Adds "Enable" and "Disable" napps methods.

### CODE REVIEW

File: kyco/controller.py [hunk 1]
```diff
@@ -564,3 +564,22 @@ def unload_napps(self):
         for napp_name in list(self.napps):
             if not isinstance(self.napps[napp_name], KycoCoreNApp):
                 self.unload_napp(napp_name)
+
+    def enable_napp(self, napp_author, napp_name):
+        """Enable a NApp by creating the needed symbolic link on the system."""
+
+        napp_abs_path = os.path.join(self.options.napps, '.installed',
```
Comment (beraldoleal) [inline kyco/controller.py]: this vars should be more global.

File: kyco/controller.py [hunk 2]
```diff
@@ -565,6 +565,18 @@ def unload_napps(self):
...
+
+        napp_sym_path = os.path.join(self.options.napps,
+                                     napp_author, napp_name)
+
+        if os.path.exists(napp_sym_path):
+            os.remove(napp_sym_path)
+            log.info('The NApp %s/%s disabled.', napp_author, napp_name)
+        else:
+            log.warning('NApp %s/%s was not enabled.', napp_author, napp_name)
+
```
Comment (beraldoleal) [inline kyco/controller.py:589]: May be is more pythonic with try... except

File: kyco/config.py [hunk 3]
```diff
@@ -16,6 +17,8 @@
 else:
     BASE_ENV = '/'
 
+# include napps modules
+sys.path.append(os.path.join(BASE_ENV, 'var/lib/kytos'))
```
Comment (beraldoleal) [inline kyco/config.py]: `os.path.join(BASE_ENV, 'var/lib/kytos')` is already been used on many parts of code. Please make this a var or config.

**Review by beraldoleal — CHANGES_REQUESTED** [review]

**Comment by cemsbr** [comment]
Note: build failed :(

**Pull request merged**